src/components/tooltip/Tooltip.tsx

```diff
@@ -170,7 +170,7 @@ const Tooltip: React.FC<TooltipProps> = ({
             ref={setPopperElement}
             className={classnames(
               type === 'tooltip' ? 'tk-tooltip' : 'tk-hint',
-              { className }
+              [className]
```

Comment:
You can use directly the 'className' instead of using an array. Because it will be ignored if it's a wrong value (undefined, ...). (See https://github.com/JedWatson/classnames)
```
className={classnames(
              type === 'tooltip' ? 'tk-tooltip' : 'tk-hint',
              className
              )}
```

Reply:
thanks. I have changed the code